features/networking/sdn.feature

```diff
@@ -258,3 +258,42 @@ Feature: SDN related networking scenarios
     When I run cmds on all ovs pods:
       | bash | -c | pgrep ovs-vswitchd |
     Then the step should succeed
+
+  #@author huirwang@redhat.com
+  #@case_id OCP-25706
+  #@bug_id 1669311
+  @admin
+  @destructive
+  Scenario: Killing ovs process should not put sdn and ovs pods in bad shape
+    Given I have a project
+    And evaluation of `project.name` is stored in the :user_project clipboard
+    Given I have a pod-for-ping in the project
+    Then evaluation of `pod.node_name` is stored in the :node_name clipboard
+    And evaluation of `pod.name` is stored in the :hello_pod clipboard
+    When I run command on the "<%= cb.node_name %>" node's sdn pod:
+      | bash | -c | pgrep ovs-vswitchd |
+    Then the step should succeed
+    When I run command on the "<%= cb.node_name %>" node's sdn pod:
+      | bash | -c | pgrep ovs-vswitchd \| xargs kill -9 |
+    Then the step should succeed
+    And I wait up to 60 seconds for the steps to pass:
+    """
+    Given I switch to cluster admin pseudo user
+    And I use the "openshift-sdn" project
+    And all pods in the project are ready
+    """
+    #Restart sdn pod
```

Comment:
why the SDN pod need to be restarted here?

Reply:
@zhaozhanqi It was from your previous comments required.
@huiran0826 how about adding one step (check the pod if work well in that node) after the sdn pod is recreated.

Reply:
> @zhaozhanqi It was from your previous comments required. Below is previous comments. If you mean sdn pod recreating by killing ovs-vswitchd, actually that will not cause sdn pod restart.
> 
> @huiran0826 how about adding one step (check the pod if work well in that node) after the sdn pod is recreated.
> @zhaozhanqi Thanks for the comments.
> Actually restarting sdn pod is not the required step in this case, if you think it is better to add a checkpoint for restarting sdn pod, I can add it.

oh, misunderstanding this,  I mean check the test pod(ping for pong) if works well after the ovs-vswitch is restarted. 
 BTW, the sdn pod did not be restarted after killing ovs-vswitched ?

Reply:
@zhaozhanqi Updated codes, please help check!